packages/client/hmi-client/src/components/widgets/tera-input.vue

```diff
@@ -82,7 +83,7 @@ onMounted(() => {
 const unmask = () => {
 	// convert back to a number when finished
 	if (isNistType && !getErrorMessage.value) {
-		emit('update:modelValue', nistToNumber(maskedValue.value));
+		emit('update:model-value', nistToNumber(maskedValue.value));
```

Comment:
Do we need to change all other places that use tera-input?

Reply:
Others were already referring to this as `model-value` I checked!